base/docs/helpdb/Base.jl

```diff
@@ -2083,11 +2083,18 @@ appended to an internal buffer of backtraces.
 :@profile
 
 """
-    extrema(itr)
+    extrema(itr) -> Tuple
```

Comment:
you need to adjust the rst signature to match this change, add the new signature to the rst, run `make docs` and commit the generated rst